quoridor-app/app/component/board.js

```diff
@@ -22,23 +23,31 @@ const generateKey = (type, position) => {
   return `${type}:${position.column}${position.row}`;
 };
 
-const Board = ({ squares, pawns }) => {
-  if (!squares) {
+const isSquareDisabled = (moves, position) => {
+  const exist = moves.filter(
+    move => move.column === position.column && move.row === position.row
+  );
+  return exist.length === 0;
+};
+
+const Board = props => {
```

Comment:
Not mandatory at all, but you destructure your props here `const Board = ({ squares, pawns }) => {`, I found it makes the code more readable